charts/managed-serviceaccount/templates/clustermanagementaddon.yaml

```diff
@@ -21,3 +19,8 @@ spec:
     defaultConfig:
       name: {{ .Values.addOnTemplateName | default (print "managed-serviceaccount-" .Chart.Version) }}
 {{- end }}
+  installStrategy:
```

Comment:
Can we also control whether to install the agent by installStrategy or not? like the`.Values.agentInstallAll` did? 
So users who want to install the agent by themselves are not affected.

Reply:
Currently, `.Values.agentInstallAll` is using `WithInstallStrategy` from the addon framework. However, this approach is deprecated, and it is recommended to use the install strategy in `ClusterManagementAddon` instead. Is there any other way to install agent on all managed clusters using the addon framework?

Reply:
using the install strategy in ClusterManagementAddon is the recommended way, no other way to install agent using the addon framework. But users should be able to decide whether to use the install strategy or install agent by themselves(create a managedclusteraddon CR in the target cluster namespace)